src/components/PokeCard.js

```diff
@@ -0,0 +1,189 @@
+import React, { useState, useRef } from "react";
+import { mockPokemonData } from "../mock/pokeData";
```

Comment:
it is better if you load mock data out of the component, then pass the value as properties

Reply:
Yes, you're right, I already did this, thank you!

App.js
```javascript
import { mockPokemonData } from "./mock/pokeData";

export default function App() {
  return (
    <div className="App">
      <h1>Welcome to your second mini challenge!</h1>
      <PokeCard mockPokemonData={mockPokemonData} />
    </div>
  );
}
```

PokeCard.js
```javascript

function PokeCard({ mockPokemonData }) {// <---- here it comes
  const [cardStyle, setCardStyle] = useState(stylesNormalCard());
  const [spriteStyle, setSpriteStyle] = useState(styleNormalSprite());
  const [textStyle, setTextStyle] = useState(styleNormalText());
  const cardRef = useRef(null);

  ...
}
```